Orange/clustering/hierarchical.py

```diff
@@ -355,13 +355,14 @@ def optimal_leaf_ordering(tree, distances, progress_callback=None):
 
     :param Tree tree:
         Binary hierarchical clustering tree.
-    :param Orange.misc.DistMatrix distances:
-        DistMatrix that was used to compute the clustering.
+    :param numpy.ndarray distances:
+        A (N, N) numpy.ndarray of distances that were used to compute
+        the clustering.
     :param function progress_callback:
         Function used to report on progress.
 
     """
-    distances = distances.X
+    distances = numpy.asarray(distances)
```

Comment:
`distances` might be a ndarray  'subclass' (for instance a numpy.matrix). If so then asarray creates an array that is a view of it's contents.